charts/authelia/values.yaml

```diff
@@ -832,27 +931,209 @@ secret:
   # labels:
   #   myLabel: myValue
 
+  mountPath: /config/secrets
+
+  ## Secrets.
   jwt:
     key: JWT_TOKEN
     # value:
+    filename: JWT_TOKEN
   ldap:
     key: LDAP_PASSWORD
     # value:
+    filename: LDAP_PASSWORD
   storage:
     key: STORAGE_PASSWORD
     # value:
+    filename: STORAGE_PASSWORD
   session:
     key: SESSION_ENCRYPTION_KEY
     # value:
+    filename: SESSION_ENCRYPTION_KEY
   duo:
     key: DUO_API_KEY
     # value:
+    filename: DUO_API_KEY
   redis:
     key: REDIS_PASSWORD
     # value:
+    filename: REDIS_PASSWORD
+  redisSentinel:
+    key: REDIS_SENTINEL_PASSWORD
+    # value:
+    filename: REDIS_SENTINEL_PASSWORD
   smtp:
     key: SMTP_PASSWORD
     # value:
+    filename: SMTP_PASSWORD
+  oidcPrivateKey:
+    key: OIDC_PRIVATE_KEY
+    # value:
+    filename: OIDC_PRIVATE_KEY
+  oidcHMACSecret:
+    key: OIDC_HMAC_SECRET
+    # value:
+    filename: OIDC_HMAC_SECRET
+
+  ## HashiCorp Vault Injector configuration.
+  vaultInjector:
+
+    ## Enable the vault injector annotations. This will disable secret injection via other means.
+    ## To see the annotations and what they do see: https://www.vaultproject.io/docs/platform/k8s/injector/annotations
+    ## Annotations with a blank string do not get configured at all.
+    ## Additional annotations can be configured via the secret.annotations: {} above.
+    ## Secrets are by default rendered in the /config/secrets directory. Changing this can be done via editing the
+    ## secret.mountPath value. You can alter the filenames with the secret.<secretName>.filename values.
+    enabled: false
+
+    ## The vault role to assign via annotations.
+    ## Annotation: vault.hashicorp.com/role
+    role: authelia
+
+    agent:
+      ## Annotation: vault.hashicorp.com/agent-inject-status
+      status: update
+
+      ## Annotation: vault.hashicorp.com/agent-configmap
+      configMap: ""
+
+      ## Annotation: vault.hashicorp.com/agent-image
+      image: ""
+
+      ## Annotation: vault.hashicorp.com/agent-init-first
+      initFirst: "false"
+
+      ## Annotation: vault.hashicorp.com/agent-inject-command
+      command: "sh -c 'kill HUP $(pidof authelia)'"
+
+      ## Annotation: vault.hashicorp.com/agent-run-as-same-user
+      runAsSameUser: "true"
```

Comment:
We must also add securityContext to the pod spec, otherwise it throws following error(which can be seen at replicaset messages): 
```
error creating new agent sidecar: No SecurityContext found for Container 0
```

something as minimal as code bellow should be sufficient
```
spec:
  securityContext:
    runAsUser: 1000
```